src/config.ts

```diff
@@ -29,6 +29,24 @@ const mainnetConfig: ClusterConfig = {
 };
 ///
 
+export const getEnvironment = () => {
```

Comment:
We can also set `SENTRY_ENVIRONMENT` in env, that seems simpler. Less code. If possible

> By default the SDK will try to read this value from the SENTRY_ENVIRONMENT environment variable (except for the browser SDK where this is not applicable).